### DISCUSSION TITLE
Including the skin in Open-Shell

### DISCUSSION THREAD

**Discussion by Henri-J-Norden** [bonzibudd/Fluent-Metro#169] [General]
This skin is fantastic and by far beats any skins Open-Shell comes with by default when it comes to a modern look or customizability. 

Have you considered merging it into Open-Shell? It would be much more convenient to not have to manually install it in addition to Open-Shell - even if e.g. it would be a few versions out of date, it would at least let you know that the skin exists.

_PS. I haven't tried Fluent-Refresh, but by tweaking the settings I got this to fit Windows 11 quite well (just imagine it being shorter if you don't like how long mine is):_ 
<img width="232" alt="clipboard" src="https://user-images.githubusercontent.com/55378880/141303444-55631de6-103d-46b3-8dd7-36ce3e650ff2.png">

**Comment by bonzibudd** [comment]
Hi, sorry for the late response, but while I think it could be very nice to have this available to users by default, I’d like to wait until I feel more confident about the codebase of Open-Shell to make a more natural integration. Also, due to the constantly changing nature of Windows 10/11, it’s hard to say the longevity of the skin, and some features have already been broken by updates. I’d like for the foundation of the program to be improved upon before I start pushing changes to the top level UI.
